test/TrafficInjector/TrafficInjector.h

```diff
@@ -34,10 +34,12 @@
 
 #include <exception/Exception.hpp>
 #include <MessageSerializer.h>
-#include <topic_types/types.h>
-#include <topic_types/typesPubSubTypes.h>
-#include <fastdds_statistics_backend/topic_types/monitorservice_types.h>
-#include <fastdds_statistics_backend/topic_types/monitorservice_typesPubSubTypes.h>
+#include <topic_types/types.hpp>
+#include <topic_types/typesPubSubTypes.hpp>
+
+#include <fastdds_statistics_backend/topic_types/monitorservice_types.hpp>
+
+#include <fastdds_statistics_backend/topic_types/monitorservice_typesPubSubTypes.hpp>
```

Comment:
Remove unnecessary line breaks
```suggestion
#include <topic_types/types.hpp>
#include <topic_types/typesPubSubTypes.hpp>
#include <fastdds_statistics_backend/topic_types/monitorservice_types.hpp>
#include <fastdds_statistics_backend/topic_types/monitorservice_typesPubSubTypes.hpp>
```